src/app/components/ThemeProvider/withThemeProvider.tsx

```diff
@@ -3,13 +3,19 @@ import { Global, ThemeProvider as EmotionThemeProvider } from '@emotion/react';
 import useIsPWA from '#app/hooks/useIsPWA';
 import focusIndicator from './focusIndicator';
 import { RequestContext } from '../../contexts/RequestContext';
-import { MEDIA_ARTICLE_PAGE, TV_PAGE } from '../../routes/utils/pageTypes';
+import {
+  LIVE_TV_PAGE,
+  MEDIA_ARTICLE_PAGE,
+  TV_PAGE,
+} from '../../routes/utils/pageTypes';
 import { BrandPalette, Typography, BrandSVG } from '../../models/types/theming';
 import { PageTypes } from '../../models/types/global';
 import getThemeConfig from './getThemeConfig';
 
 const isDarkUiPage = (pageType: PageTypes) =>
-  pageType === MEDIA_ARTICLE_PAGE || pageType === TV_PAGE;
+  pageType === MEDIA_ARTICLE_PAGE ||
+  pageType === TV_PAGE ||
+  pageType === LIVE_TV_PAGE;
```

Comment:
We can now make this more readable:

```suggestion
const isDarkUiPage = (pageType: PageTypes) =>
 [MEDIA_ARTICLE_PAGE, TV_PAGE, LIVE_TV_PAGE].includes(pageType);
```